numpy/_core/fromnumeric.pyi

```diff
@@ -8,7 +9,7 @@ from numpy import (
     int64,
     intp,
     float16,
-    bool_,
+    bool,
```

Comment:
The diff would be a lot smaller if you'd leave out this import. Using the builtin can then stay unchanged. Seems preferable to me to use `np.bool` (unless that doesn't work?).